modules/shared/programs/claude/files/skills/plan-with-questions/references/resume-state.md

```
@@ -131,35 +131,35 @@ echo "branch=$BRANCH, HEAD=$HEAD_ANCHOR, dirty=$DIRTY"
+- 체크박스 evidence: 완료 체크박스(`- [x]`)는 evidence 또는 validation note 없이 전환 금지. plan에 "Step 5 DA 완료"라 적으려면 외부 검토 verdict 요약, stable artifact name, 또는 validation note가 함께 있어야 한다. ephemeral scratch reference를 durable evidence로 쓰지 않는다.
+- Last Updated 동기화: `Last Updated`가 바뀌면 `Change Log`도 같은 날짜로 갱신된 entry가 있어야 한다.
+- Mode 전환은 DL: `Mode` 필드가 바뀌면(`for_action` → `for_prd`) 반드시 Decision Log에 기록한다.
+- Baseline drift 처리 의무: Baseline drift 또는 dirty 상태 ambiguity 감지 시 Step 1-2 재실행 또는 사용자 확인이 필요하다. 메인 LLM 자체 판단으로 "drift는 무시해도 되겠다"며 `Resume From`으로 점프하지 않는다.
 
 ## 재개 호출 패턴 (사용자 입장)
 
 다음 세션에서 사용자가 다음 중 하나로 시작하면 재개:
 
-- 명시적: `"`.claude/plans/<path>.md` Resume From부터 이어가자"` 
-- plan 파일 path만: `".claude/plans/<path>.md"` (메인 LLM이 Status / Resume From 읽고 판단)
+- 명시적: `"`.claude/plans/<path>.md` Resume From부터 이어가자"`
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)